bitnami/etcd/templates/scripts-configmap.yaml

```diff
@@ -181,6 +181,13 @@ data:
         fi
     fi
 
+    if [[ $(stat -c "%a" "$ETCD_DATA_DIR") != *700 ]]; then
+        echo "==> Setting data directory permissions to 700 in a recursive way (required in etcd >=3.4.10)" 1>&3 2>&4
+        chmod -R 700 $ETCD_DATA_DIR
+    else
+        echo "==> The data directory is already configured with the proper permissions" 1>&3 2>&4
+    fi
+
```

Comment:
Add the block before line 122

Reply:
In that case, we can not ensure that the data directory exists at this point:
```
stat: cannot stat '/bitnami/etcd/data': No such file or directory
==> Setting data directory permissions to 700 in a recursive way (required in etcd >=3.4.10)
chmod: cannot access '/bitnami/etcd/data': No such file or directory
```
we can place the block in the else of `if [[ ! -d "$ETCD_DATA_DIR" ]]; then` if we want to ensure that the data directory is created from a previous deployment or as it's right now that is going to check everything (previous deployments and created in this one)